src/integrations/third-party/elementor.php

```diff
@@ -174,6 +172,9 @@ public function register_elementor_hooks() {
 	 * @return void
 	 */
 	public function init() {
+		$this->social_is_enabled            = $this->options->get( 'opengraph', false ) || $this->options->get( 'twitter', false );
```

Comment:
@Xyfi Initializing `$this->social_is_enabled` here instead of in the constructor will break the `if ( $this->social_is_enabled ) {` check in `save_postdata()` as that function will be called via AJAX and the class will not have been constructed by then and the `$this->social_is_enabled` will equal to `null`. 

With the live version, `$this->social_is_enabled` equals to `true|false` in that check.